interface.rb

```diff
@@ -0,0 +1,28 @@
+module Interface
+  def greeting
+    p 'Welcome to our game, kid. I have thought of a number. Can you guess what the number is?'
+    p 'How many attempts do you need to win the game?'
+  end
+
+  def counter
+    p "You have #{@current_attempts -= 1} attempt(s) left!"
+  end
+
+  def won_game(hiden_number)
+    p 'Congrats! You won the game, kid!' if hiden_number == '++++'
+  end
+
+  def game_over(hiden_number)
+    p 'Game over' if hiden_number != @number && @current_attempts.zero?
```

Comment:
what is @number ? And it's bad idea to use instance variable insed module, and why did you decide to use a module?

Reply:
Fixed